multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -157,6 +198,13 @@ write_files:
           "http://127.0.0.1:8080/api/v1/namespaces/kube-system/services"
       done
 
+  - path: /opt/bin/host-rkt
```

Comment:
Is there an issue we can reference that describes why this is necessary? If not maybe add some comments - I already don't remember.